package.json

```diff
@@ -90,6 +92,7 @@
     "chalk": "2.4.2",
     "cheerio": "1.0.0-rc.3",
     "coffee-react": "5.0.1",
+    "coffeescript": "2.4.1",
```

Comment:
for some reason, `coffeescript` dependency was not found by default, even if it was a dependency of `@cypress/browserify-preprocessor`

Reply:
I'm confused why we need this. There is no coffeescript in this repo at all. This a dependency of @cypress/browserify-preprocessor??